### PR TITLE
Cross validation code

### PR DESCRIPTION
**Pull request by mergu**

### CODE REVIEW

**Comment by mergu** [comment]
This is ready for review

File: code/overview/cross-validation/holdout.py [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+# Create a pandas DataFrame from the diabetes dataset
+dataframe = pandas.DataFrame(dataset.data, columns=columns)
+
+# Split via the holdout method
+x_train, x_test, y_train, y_test = train_test_split(
+  dataframe, dataset.target, train_size=TRAIN_SPLIT, test_size=1-TRAIN_SPLIT)
+
+# Print our test and training data
+print("X Test Data:")
+print(x_test)
```
Comment (b-sherman) [inline code/overview/cross-validation/holdout.py]: The code looks well-documented. I ran it to check how the output looks and it's a bit confusing as to what you're trying to convey. If you're just trying to show how the data was split, providing output like "Number of entries in test data" and "Number of entries in training data" with len(x_test) and len(x_train) might be sufficient. Throwing in percentages or the total number of entries in the dataset would also help with that.

File: code/overview/cross-validation/k-fold.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# An example of K-Fold Cross Validation split
+
+import numpy
+from sklearn.model_selection import KFold
+
+# Configurable constants
+NUM_SPLITS = 3
+
+
+def print_result(kfold):
```
Comment (b-sherman) [inline code/overview/cross-validation/k-fold.py]: The documentation is good. Output is pretty nice but the '\t' has some display issues depending on where it is used in the line. Just using a '\n' will still differentiate between the training and testing set and appear more consistent. An additional newline and label with fold number would also be helpful to differentiate between folds.

File: code/overview/cross-validation/leave-p-out.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Example of LOOCV and LPOCV splitting
+
+import numpy
+from sklearn.model_selection import LeaveOneOut, LeavePOut
+
+# Configurable constants
+P_VAL = 2
+
+
+def print_result(method):
```
Comment (b-sherman) [inline code/overview/cross-validation/leave-p-out.py]: The documentation looks good. I would suggest the same sort of changes that I mentioned in k-fold.py for consistency. That being said, the tab issues don't seem to present themselves with any of the reasonable P_VAL values so it might be fine to leave as is.

**Review by b-sherman — CHANGES_REQUESTED** [review]

**Review by b-sherman — APPROVED** [review]

**Review by astorfi — CHANGES_REQUESTED** [review]
Thank you so much, guys ... It looks good ... I just have few final suggestions and wrote them in slack. Please change your documenation accordingly. Thanks

**Review by astorfi — APPROVED** [review]
